executor/stale_txn_test.go

```diff
@@ -193,62 +178,7 @@ func (s *testStaleTxnSerialSuite) TestStalenessAndHistoryRead(c *C) {
 	tk.MustExec("commit")
 }
 
-func (s *testStaleTxnSerialSuite) TestStalenessTransactionSchemaVer(c *C) {
```

Comment:
I think we should add a testcase that requesting `stale transaction` after the ddl job finish while transaction still use the stale infoschema.

Reply:
It is added back.